lib/manageiq/cross_repo/runner.rb

```diff
@@ -67,6 +70,10 @@ def system!(*args)
       exit($?.exitstatus) unless $?.success?
     end
 
+    def script_source
+      Base.descendants.detect(&:available?)
```

Comment:
Obvious comment is obvious, but we'll have to make sure not to have an old .travis.yml when we switch over to github actions

Reply:
I think it's ok either way...the goal is to run the tests...if they both "work" then it will pick the first one.